ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -2246,3 +2246,33 @@ def test_numpy_instance_len__(
         frontend=frontend,
         frontend_method_data=frontend_method_data,
     )
+
+# __array__
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="numpy.array",
+    method_name="__array__",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+    ),
+)
+def test_numpy_instance_array__(
```

Comment:
LGTM! Please resolve the conflicting files, and the PR will be ready to merge! Thank you for your contribution 😊

Reply:
@ZoeCD,  @guodh321 Line `2259` of ```ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py``` does not look right.

Reply:
Hi @danny-1k , I found some lines got lost when merging into the master, could you or your colleagues add these below? Thanks.
    dtype_and_x,
    frontend_method_data,
    init_flags,
    method_flags,
    frontend,
):
    input_dtypes, x = dtype_and_x
    helpers.test_frontend_method(
        init_input_dtypes=input_dtypes,
        init_all_as_kwargs_np={
            "object": x[0],
        },
        method_input_dtypes=input_dtypes,
        method_all_as_kwargs_np={},
        init_flags=init_flags,
        method_flags=method_flags,
        frontend=frontend,
        frontend_method_data=frontend_method_data,
    )